suites/squid/rados/deploy-stretch-cluster-mode.yaml

```diff
@@ -0,0 +1,196 @@
+
+  - test:
+      name: OSD deployment
+      desc: Add OSD services using spec file.
+      module: test_cephadm.py
+      polarion-id: CEPH-83573746
+      config:
+        steps:
+          - config:
+              command: apply_spec
+              service: orch
+              validate-spec-services: true
+              specs:
+                - service_type: osd
+                  service_id: stretch_osds
+                  encrypted: "true"                     # boolean as string
+                  placement:
+                    nodes:
+                      - mero005
+                      - mero007
+                      - mero009
+                      - mero011
+                      - mero013
+                      - mero014
+                      - mero017
+                      - mero018
+                      - mero019
+                      - mero020
+                  spec:
+                    data_devices:
+                      paths:
+                        - /dev/sdc
+                        - /dev/sdd
+                        - /dev/sde
+                        - /dev/sdf
+                        - /dev/sdg
+                        - /dev/sdh
+                    db_devices:
+                      paths:
+                        - /dev/nvme0n1
+  - test:
+      name: MDS Service deployment with spec
+      desc: Add MDS services using spec file
+      module: test_cephadm.py
+      polarion-id: CEPH-83574728
+      config:
+        steps:
+          - config:
+              command: shell
+              args:               # arguments to ceph orch
+                - ceph
+                - fs
+                - volume
+                - create
+                - cephfs
+          - config:
+              command: apply_spec
+              service: orch
+              validate-spec-services: true
+              specs:
+                - service_type: mds
+                  service_id: cephfs
+                  placement:
+                    label: mds
+
+  - test:
+      name: RGW Service deployment
+      desc: RGW Service deployment
+      module: test_cephadm.py
+      polarion-id: CEPH-83574728
+      config:
+        steps:
+          - config:
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.1
+              args:
+                placement:
+                  label: rgw
+
+  - test:
+      name: Configure clients
+      desc: Configures multiple client nodes on cluster
+      module: test_parallel.py
+      parallel:
+        - test:
+            name: Configure client 1
+            desc: Configures client.1 admin node on cluster
+            module: test_client.py
+            polarion-id:
+            config:
+              command: add
+              id: client.1                      # client Id (<type>.<Id>)
+              nodes:
+                  - node10:
+                      release: 6
+                  - node11:
+                      release: 6
```

Comment:
6 -> 8

Reply:
Made the changes as per the comments.

Reply:
Same as above. Please use the GA'ed versions for custom clients.

Reply:
Changed the GA version to 7